app/views/talks/_talk.json.jbuilder

```diff
@@ -0,0 +1,53 @@
+json.cache! talk do
+  json.date talk.date
+  json.description talk.description
+  json.kind talk.kind
+  json.original_title talk.original_title
+  json.slides_url talk.slides_url
+  json.slug talk.slug
+  json.summary talk.summary
+  json.title talk.title
+  json.video_id talk.video_id
+  json.video_provider talk.video_provider
+
+  if talk.event.present?
+    json.event do
+      json.slug talk.event.slug
+      json.name talk.event.name
+      json.start_date talk.event.start_date
+      json.end_date talk.event.end_date
+
+      if talk.event.organisation.present?
+        json.organisation do
+          json.id talk.event.organisation.id
+          json.name talk.event.organisation.name
+          json.slug talk.event.organisation.slug
+        end
+      end
+    end
+  end
+
+  json.speakers talk.speakers do |speaker|
+    json.id speaker.id
+    json.name speaker.name
+    json.slug speaker.slug
+    if speaker.user.present?
+      json.bio speaker.user.bio
+      json.avatar_url speaker.user.avatar_url
+    end
+  end
+
+  json.topics talk.approved_topics do |topic|
+    json.id topic.id
+    json.name topic.name
+    json.slug topic.slug
+  end
+
+  json.transcript do
+    json.raw talk.raw_transcript
+    json.enhanced talk.enhanced_transcript
+  end
```

Comment:
Those can be pretty large fields. before this PR it was only used for the Talks#show view. 
Now the problem is that for a speaker with lots of Talk (hello Marco/Aaron) it generates a huge json. 

I think we should remove the transcript when we fetch a collection of talks